store/postgres/src/dynds.rs

```diff
@@ -154,6 +154,11 @@ pub(crate) fn insert(
         })
         .collect::<Result<_, _>>()?;
 
+    if dds.is_empty() {
```

Comment:
It would be better to check `data_sources` before even constructing `dds`.

Reply:
Collecting an empty iterator is a no-op anyways, but I suppose it's more readable if the check is at the start of the function.